gcloud-java-dns/src/test/java/com/google/gcloud/dns/it/ITDnsTest.java

```diff
@@ -962,4 +964,26 @@ public void testListDnsRecords() {
       clear();
     }
   }
+
+  @Test
+  public void testListZoneBatch() {
+    DNS.create(ZONE1);
+    DNS.create(ZONE_EMPTY_DESCRIPTION);
+    try {
+      DnsBatch batch = DNS.batch();
+      DnsBatchResult<Page<Zone>> batchResult = batch.listZones();
+      batch.submit();
+      assertTrue(batchResult.submitted());
+      Iterator<Zone> iteratorBatch = batchResult.get().iterateAll();
+      Iterator<Zone> iteratorList = DNS.listZones().iterateAll();
```

Comment:
Is zone list order guaranteed?  Perhaps you should do some sort of set comparison instead if not.